presence-detector.py

```diff
@@ -40,11 +40,16 @@ def __init__(self, config_file):
         self.clients_seen = {}
 
     def ha_seen(self, client, seen=True):
-        location = "home"
+        location = self.settings.zone
```

Comment:
A default for this setting would be nice, since this will break existing setup that don't have the 'zone' parameter.